src/environments/environment.ts

```diff
@@ -0,0 +1,24 @@
+export const environment = {
+  production: false,
+  port: 3000,
+  api: {
+    prefix: "/api"
+  },
+  db: {
+    database: 'nodeJs',
```

Comment:
Avoid to use capitall letters in database names

Reply:
Done.